changelog/0.8.2/issue-1497

```diff
@@ -0,0 +1,7 @@
+Enhancement: Add --read-data-subset flag to check command
```

Comment:
This file needs to be moved to `changelog/0.8.3`, we've released 0.8.2 yesterday :)